scripts/dotnet-cli-build/TestTargets.cs

```diff
@@ -96,10 +96,9 @@ public static BuildTargetResult RestoreTestAssetProjects(BuildTargetContext c)
                 .Execute()
                 .EnsureSuccessful();
 
-            // The 'ProjectWithTests' is a portable test app. Cannot call --infer-runtimes on it, since on win x64 machines,
-            // the x86 runtime is being inferred, and there are no x86 DotNetHost packages
             dotnet.Restore(
```

Comment:
I went ahead and moved this, changed the tests, and removed where it was getting built separately.